library/std/src/io/buffered/mod.rs

```diff
@@ -17,6 +18,9 @@ pub use bufwriter::BufWriter;
 pub use linewriter::LineWriter;
 use linewritershim::LineWriterShim;
 
+#[unstable(feature = "stdout_switchable_buffering", issue = "78515")]
```

Comment:
Any idea why this requires a stability attribute but the other reexports (like `pub use linewriter::LineWriter`) don't?